index.html

```diff
@@ -170,73 +170,75 @@ <h4 class="topic"  data-target-tab-panel-id="about">
         <div class="container tabs-section main-page-tab-panel" data-tab-panel-id="quickstart">
             <h5 class="pt-4 pb-2 lh-lg"><span id="qtLabel">Choose from some of ESP's pre-built, out-of-the-box examples to flash and play</span></h5>
             <div class="form">
-                <div class="field-container">
-                    <label>
-                        Select Application
-                    </label>
-                    <div class="field">
-                        <select name="device" id="device">
-                            <option value="Matter-AllClustersApp">Matter-AllClustersApp</option>
-                            <option value="RainMaker-Fan">RainMaker-Fan</option>
-                            <option value="RainMaker-GPIO">RainMaker-GPIO</option>
-                            <option value="RainMaker-HomekitSwitch">RainMaker-HomekitSwitch</option>
-                            <option value="RainMaker-LedLight">RainMaker-LedLight</option>
-                            <option value="RainMaker-MultiDevice">RainMaker-MultiDevice</option>
-                            <option value="RainMaker-Switch">RainMaker-Switch</option>
-                            <option value="RainMaker-TemperatureSensor">RainMaker-TemperatureSensor</option>
-                        </select>
-                    </div>
-                </div>
-                
-                <div class="field-container">
-                    <label>
-                        ESP Chipset Type
-                    </label>
-                    <div class="field">
-                        <div id="chipsets">
-                            <div class="form-check-inline">
-                                <label class="form-check-label" for="radio1">ESP32&nbsp;
-                                    <input type="radio" checked class="form-check-input" name="chipType" id="radio1" value="esp32_matter_all-clusters-app_te6_merged.bin">
-                                </label>
+                <div class="d-flex justify-content-between flex-wrap form-config-info-container-wrapper">
+                    <div class="form-container">
+                        <div class="field-container">
+                            <label>
+                                Select Application
+                            </label>
+                            <div class="field">
+                                <select name="device" id="device">
+                                    <option value="Matter-AllClustersApp">Matter-AllClustersApp</option>
```

Comment:
I think we dynamically populate this based on the apps configured in the toml file. So we can get rid of these default hardcoded options

Reply:
Removed the default hardcoded options in the latest commit